soc/espressif/esp32c2/Kconfig

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_ESP32C2
+	select RISCV
+	select RISCV_GP
+	select DYNAMIC_INTERRUPTS
+	select CLOCK_CONTROL
+	select PINCTRL
+	select RISCV_ISA_RV32I
+	select RISCV_ISA_EXT_A
```

Comment:
I think C2 has no Atomic set. PLease confirm and remove `select RISCV_ISA_EXT_A`.